tests/EmitTest.elm

```diff
@@ -45,6 +45,15 @@ javascript =
                     , ( "negative int", Literal (Int -998), "-998" )
                     ]
                 )
+            , describe "Float"
+                (List.map runTest
+                    [ ( "positive float", Literal (Float 12.3), "12.3" )
+                    , ( "negative float", Literal (Float -12.3), "-12.3" )
+                    , ( "scientific notation", Literal (Float 1e2), "100" )
+                    , ( "negative scientific notation", Literal (Float -1e2), "-100" )
+                    , ( "negative exponent", Literal (Float 5e-2), "0.05" )
```

Comment:
I don't think we need to concern ourselves with scientific notation (these last three tests) in emit tests.

If eg. `1e2` passes the parser, it goes into the rest of the pipeline as `100` ... so the test *doesn't* test anything about scientific notation, I believe. It is definitely useful as a parser test though! :+1: